deployment/docker-compose.integration.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.4"
```

Comment:
Should we use a newer version? 3.4 seems quite old

Reply:
I've copied it from Rasa X, but sure